src/pages/Rating/Components/MovieInfo.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux';
+import { actionCreators } from '../../../store';
+import CommentWrite from "./CommentWrite";
+import { pinkFlag, normalFlag, blueEye, normalEye, talkIcon, banIcon } from './Datas.js'
+
+class MovieInfo extends Component {
+
+  handleModal = () => {
+    this.props.ChangeMovieInfo(false)
+    this.props.ChangeCommentWrite(false)
+  }
+
+  handleClickOutside = ({target}) =>  {
+    if(target.className==="MovieInfo"|| target.className==="close_btn") {
+      this.handleModal();
+      this.props.CommentValueChange('')
+    }
+  }
+
+  componentDidMount() {
+    window.addEventListener('click', this.handleClickOutside)
+  }
+  componentWillUnmount() {
+    window.removeEventListener('click', this.handleClickOutside)
+  }
+  
+  render() {
+    const { name, img, date, status } = this.props.currentMovie;
+    const { isCommentWrite, ChangeCommentWrite } = this.props
+    return (
+      <div className="MovieInfo">
+        <div className="movie_info_wrap">
+          <div className="movie_info_header">
+            <div className="info_image">
+          <span style={{ backgroundImage: `url(${img})`}}></span>
+            </div>
+          <div className="info_desc">
+            <div>{name}</div>
+            <span>영화・{date}</span>
+          </div>
+          </div>
+          <div className="movie_info_icon">
+            <div>
+              {status === 'wished' 
+              ? <img src={pinkFlag} alt=""/>
+              : <img src={normalFlag} alt=""/>
+              }
```

Comment:
```suggestion
               <img src={status === "watching" ? blueEye : normalEye} alt="" />
```
위와 같이 적으면 한 줄로 표현이 가능하겠네요! alt 속성도 value 값을 적어주세요!